packages/jamulus/PKGBUILD

```diff
@@ -5,39 +5,39 @@
 
 pkgbase=jamulus
 pkgname=(jamulus jamulus-headless)
-pkgver=3.8.2
-pkgrel=5
+pkgver=3.9.0beta1
+pkgrel=1
 pkgdesc="Internet jam session software"
 arch=(x86_64 aarch64)
 url='https://jamulus.io/'
 license=(GPL2 BSD custom:STK)
-depends=(gcc-libs qt5-base)
+depends=(gcc-libs qt6-base)
 provides=(jamulus-server)
 conflicts=(jamulus-git)
-makedepends=(jack)
+makedepends=(jack qt6-multimedia)
 groups=(pro-audio)
 source=("$pkgbase-$pkgver.tar.gz::https://github.com/jamulussoftware/$pkgbase/archive/r${pkgver//./_}.tar.gz"
   'jamulus.service'
   'jamulus.sysusers')
-sha256sums=('4c323db21c896c711f726319b40114d1a0b5a374fd25eb43d9af52c19fc3d55e'
+sha256sums=('0da4032277802c61c9758ddb4eea67f8636dd636d985d3250cb627d48b97ebff'
             '98e45f7f877dbc9f8113d63b6e009ff1025e73e1cce86f671b57474a4764e11f'
             '4117ad3a93b3211f679f93794b308ad292d1799a86f85a6b353cfdff8515e2f9')
 _pkgsrc=$pkgbase-r${pkgver//./_}
 
 build() {
   cd $_pkgsrc
-  qmake "CONFIG+=nosound headless" TARGET=jamulus-headless
+  qmake6 "CONFIG+=serveronly headless" TARGET=jamulus-headless
   make clean
   make
-  qmake "CONFIG+=noupcasename"
+  qmake6 "CONFIG+=noupcasename"
   make clean
   make
```

Comment:
I don't if it is good for our usage but, the COMPILING.md file from Jamulus repository says:
`make distclean
qmake6
make`
there is no need to `make clean` after qmake since `make distclean` is done before.

Reply:
IIRC there were issues when recompiling, since we're compiling one variant after another

Reply:
Same here with `make distclean` and using `makepkg`.